catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/transformer/CswRecordMapperFilterVisitor.java

```diff
@@ -262,17 +257,8 @@ public Object visit(PropertyName expression, Object extraData) {
   public Object visit(PropertyIsBetween filter, Object extraData) {
     Expression expr = visit(filter.getExpression(), extraData);
 
-    AttributeType type =
-        attributeTypes.get(((PropertyName) filter.getExpression()).getPropertyName());
-
-    LiteralExpressionImpl typedLowerExpression = (LiteralExpressionImpl) filter.getLowerBoundary();
-    setExpressionType(type, typedLowerExpression);
-
-    LiteralExpressionImpl typedUpperExpression = (LiteralExpressionImpl) filter.getUpperBoundary();
-    setExpressionType(type, typedUpperExpression);
```

Comment:
Several of these similar code blocks were removed because the data type is now correctly determined when visiting the literal expression initially in the `visit(Literal expression, Object extraData)` method.